patterns/1-initial/secure-innersource.md

```diff
@@ -0,0 +1,144 @@
+# Title
+
+Secure InnerSource
```

Comment:
Naming things is hard, which is also true for patterns :)

Especially when using the term **InnerSource** in pattern titles we want to be careful, as otherwise all InnerSource patterns end up with that term in the title.

Some alternative proposals:
- Security and InnerSource
- Satisfying Security Concerns
- Addressing Security Concerns
- Security Concerns and Mitigation
- Working with Security
- Security and Transparency
- Security and Discoverability
- Secure Discoverability
- Discoverability in a Secure Way

Titles are often named either after the **problem** they address, or the **solution** they propose.

Anybody got an idea for a great title? Please comment.

Reply:
Balancing Openness and Security

Reply:
Secure Code Sharing

Reply:
Repository Sharing Levels

Reply:
I am opting for the title "Balancing Openness and Security" for now.
Will also add some of the alternative proposals here to the "Alias" section of the pattern.

When we level up the pattern to higher states of maturity, e.g. by getting more orgs to confirm how they are using this pattern, we can reconsider what the best name for this is.

And yes, naming is hard :)